trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Lift/compute_max_lift_coeff.py

```diff
@@ -78,11 +78,11 @@ def compute_max_lift_coeff(vehicle,conditions=None):
         sweep      = wing.sweeps.quarter_chord
         sweep_deg  = wing.sweeps.quarter_chord / Units.degree # convert into degrees
         taper      = wing.taper
-        flap_chord = wing.flaps.chord
-        flap_angle = wing.flaps.angle
-        slat_angle = wing.slats.angle
-        Swf        = wing.areas.affected  #portion of wing area with flaps
-        flap_type  = wing.flaps.type
+        flap_chord = wing.control_surfaces['flap'].chord_fraction # correct !!! 
```

Comment:
Why did you switch these to brackets?

Reply:
Long story short - the control surface definition before was not general and a case by case basis - several files defined them differently. In this particular file, the key 'flap' referenced a class, like what we do in other places with 'main_wing'. 

I still went ahead and changed it. I also removed some code that was at the bottom of this script which was actually not being used,